src/components/Inputs/Schemas.module.css

```diff
@@ -2,64 +2,45 @@
     display: flex;
     flex-direction: column;
 }
+
 .container > div:nth-child(1) > div {
     margin-right: 15px;
 }
-form .formControl {
-    margin-top: 10px;
-    display: flex;
-    flex-direction: column;
-    height: 1100px;
-    flex-wrap: wrap;
-}
 
-form .formLabel {
-    margin-bottom: 5px;
+.controls {
+    display: flex;
+    flex-direction: row;
+    margin: 15px 0;
     width: 100%;
-    user-select: none;
-    color: #000;
-    font-size: 13px;
-    font-weight: 600;
-    text-transform: uppercase;
 }
 
 .group {
     width: 30%;
     margin-bottom: 10px;
 }
 
-.group label {
-    font-size: 14px;
-}
-
 .group .schema {
     margin-top: 5px;
-    display: flex;
-    flex-wrap: wrap;
-    flex-direction: row;
 }
 
-.group .schema label {
-    height: 24px;
-    width: 33%;
-    margin-right: 0;
-    align-items: flex-start;
-    white-space: nowrap;
-}
-.group .schema > div {
-    width: auto;
-    font-size: 14px;
-    align-items: flex-start;
-    line-height: inherit;
+form .formLabel {
+    margin-bottom: 5px;
+    width: 100%;
+    user-select: none;
+    color: #000;
+    font-size: 13px;
+    font-weight: 600;
+    text-transform: uppercase;
 }
 
-.group .schema > div label {
-    margin-left: 4px;
+form .formControl {
```

Comment:
I think here `column-count: 2;` would convey your intentions as well (if I assume correctly that you want to render checkboxes in two columns past a certain width). Without the risk of overflowing the container if there's too much content to fit in two 1100 pixels high columns.

Reply:
I didn't write this code.. But the way it works now, there can also be three columns.
Although I really don't like the way this is solved because all instances with a higher item amount than could fit into three cols would looks not good here.

I think we should @cooper-joe about how to do this properly..
(@cooper-joe All the available meta data schemas are listed as checkboxes here, so there could be a lot more than currently displayed at some point... But @varl said we shouldn't put time into refactoring the import-export app now. I was just working on this while waiting for progress on another import-export-app jira issue)

Reply:
> I didn't write this code.. 

Ah I see, the diff made me think this was new code.

> But the way it works now, there can also be three columns.
Although I really don't like the way this is solved because all instances with a higher item amount than could fit into three cols would looks not good here.

Yeah exactly, and for me those overflowed the container. To me it makes sense to just use 2 cols (maybe responsively add more, or less)

> But @varl said we shouldn't put time into refactoring the import-export app now. I was just working on this while waiting for progress on another import-export-app jira issue

Ah ok, makes sense 👍

Reply:
Just checked, column-count doesn't work here as it's ignoring the div and divides the last group into two columns, I'll keep the current solution once we've found something better